src/com/connectsdk/discovery/DiscoveryManager.java

```diff
@@ -87,8 +87,21 @@
  */
 public class DiscoveryManager implements ConnectableDeviceListener, DiscoveryProviderListener, ServiceConfigListener {
 
+    /**
+     * Describes a pairing level for a DeviceService. It's used by a DiscoveryManager and all
+     * services.
+     */
     public enum PairingLevel {
+        /**
+         * Indicates that pairing is off. DeviceService will never try to pair with a first
```

Comment:
A note: "Indicates that pairing is off" – out of context (`DiscoveryManager`), it implies that this value is set by something else, not the developer. Probably, "Specifies/instructs/requests that pairing should be off (should not be attempted)"?